PureInvoke/Functions/Invoke-AdvApiLookupPrivilegeValue.ps1

```diff
@@ -0,0 +1,56 @@
+
+function Invoke-AdvApiLookupPrivilegeValue
+{
+    <#
+    .SYNOPSIS
+    Calls the advapi32.dll library's `LookupPrivilegeValue` function to lookup a privilege's local unique identifier.
+
+    .DESCRIPTION
+    The `Invoke-AdvApiLookupPrivilegeValue` function calls the advapi32.dll library's `LookupPrivilegeValue` function to
+    lookup a privilege's LUID from its name. Pass the privilege's name to the `Name` parameter. If the privilege exists,
+    its LUID is returned. Otherwise nothing is returned and an error is written.
+
+    Privilege names are case-sensitive.
+
+    Also, privilege names *do not* include account rights, even though the names look similar. The following [known
+    account rights](https://learn.microsoft.com/en-us/windows/win32/secauthz/account-rights-constants) are not supported
+    by `LookupPrivilegeValue`:
+
+    * SeBatchLogonRight
+    * SeDenyBatchLogonRight
+    * SeDenyInteractiveLogonRight
+    * SeDenyNetworkLogonRight
+    * SeDenyRemoteInteractiveLogonRight
+    * SeDenyServiceLogonRight
+    * SeInteractiveLogonRight
+    * SeNetworkLogonRight
+    * SeRemoteInteractiveLogonRight
+    * SeServiceLogonRight
+    * SeUnsolicitedInputPrivilege
+
+    .EXAMPLE
+    Invoke-AdvapiLookupPrivilegeName -Name SeDebugPrivilege
+
+    Demonstrates how to call this function.
+    #>    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [String] $Name,
+
+        [String] $ComputerName

```

Comment:
Should this parameter be documented in the Description?

Reply:
Good idea. Found some other documentation improvements to make.